python/external_task_api_client/external_task_api_client_service.py

```diff
@@ -0,0 +1,71 @@
+import aiohttp
+import asyncio
+import json
+
+
+class ExternalTaskApiClientService:
+
+    def __init__(self, baseUrl):
+        self.__baseUrl = baseUrl
+
+    async def extendLock(self, identity, workerId, externalTaskId, additionalDuration):
+        uri = "task/" + externalTaskId + "/extend_lock"
```

Comment:
```suggestion
        uri = 'task/{}/extend_lock'.format(external_task_id)
```

So würde man das "pythonic" machen.